helpers/error_helper.go

```diff
@@ -0,0 +1,159 @@
+package helpers
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/leechongyan/Studtor_backend/constants"
+)
+
+type RequestError struct {
+	StatusCode int
+
+	Err error
```

Comment:
bro error is an interface that consists of 1 method, `Error() string`; this does make sense if RequestError is supposed to be _wrapping_ some underlying error and, depending on the underlying error, cause the handler to return a string accordingly.

not saying this is wrong, just wanted to make sure you're aware. can refer [here](https://golang.org/pkg/errors/) for more info

Reply:
also note that by definition, this `RequestError` **cannot** hold a string - i.e., it's a wrapper over an underlying error and injects a status code. 

if we do want a error wrapper pattern, this should take string also

Reply:
Not sure about the error wrapper pattern, maybe u can raise an issue up and we can deal with that again

Reply:
sure, the golang blog has a good article [here](https://blog.golang.org/go1.13-errors)